src/runner/main.cpp

```diff
@@ -300,37 +300,44 @@ void cleanup_updates()
         return;
     }
 
-    // Msi and exe files
-    for (const auto& entry : std::filesystem::directory_iterator(updating::get_pending_updates_path()))
+    auto update_dir = updating::get_pending_updates_path();
+    if (std::filesystem::exists(update_dir))
     {
-        auto entryPath = entry.path().wstring();
-        std::transform(entryPath.begin(), entryPath.end(), entryPath.begin(), ::towlower);
-
-        if (entryPath.ends_with(L".msi") || entryPath.ends_with(L".exe"))
+        // Msi and exe files
+        for (const auto& entry : std::filesystem::directory_iterator(updating::get_pending_updates_path()))

```

Comment:
Perhaps use ` update_dir`  here instead of `updating::get_pending_updates_path()`

Reply:
I thought I did that :D

Reply:
done